### PR TITLE
Fix/plan compare

### PR DESCRIPTION
**Pull request by Shylock-Hg**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Description:
Fix the plan comparing in tck. Support to compare the nested dictionary in plan.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by jievince — APPROVED** [review]
Well done! Thanks for fixing it.

**Review by Aiee — DISMISSED** [review]

File: tests/common/plan_differ.py [hunk 1]
```diff
@@ -2,8 +2,11 @@
 #
 # This source code is licensed under Apache 2.0 License.
 
+from codecs import ignore_errors
```
Comment (Aiee) [inline tests/common/plan_differ.py:5]: Why do we need this?

File: tests/common/plan_differ.py [hunk 2]
```diff
@@ -2,8 +2,11 @@
...
 import re
 import json
+import deepdiff
```
Comment (Aiee) [inline tests/common/plan_differ.py:8]: Is this for debugging?

Comment (Shylock-Hg) [reply tests/common/plan_differ.py:8]: Not, to compare the dict

**Review by nevermore3 — APPROVED** [review]

File: tests/common/plan_differ.py (hunk 1 above)
Comment (Shylock-Hg) [reply tests/common/plan_differ.py:5]: Already removed

**Review by Aiee — APPROVED** [review]

**Pull request merged**